slim/train_image_classifier.py

```diff
@@ -457,12 +457,21 @@ def clone_fn(batch_queue):
       # Specify the loss function #
       #############################
       if 'AuxLogits' in end_points:
-        tf.losses.softmax_cross_entropy(
-            logits=end_points['AuxLogits'], onehot_labels=labels,
-            label_smoothing=FLAGS.label_smoothing, weights=0.4, scope='aux_loss')
+            logits = end_points['AuxLogits']
+            tf.losses.softmax_cross_entropy(
+                logits=logits, onehot_labels=labels,
+                label_smoothing=FLAGS.label_smoothing, weights=0.4, scope='aux_loss')
       tf.losses.softmax_cross_entropy(
-          logits=logits, onehot_labels=labels,
-          label_smoothing=FLAGS.label_smoothing, weights=1.0)
+            logits=logits, onehot_labels=labels,
+            label_smoothing=FLAGS.label_smoothing, weights=1.0)
```

Comment:
You still changed the behavior of this code when the above `if` is `True`. Are you suggesting that the code was incorrect previously?